docs/reference/connectors/snowflake/index.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Snowflake
+sidebar_position: 1
+description:
+  "Learn how to configure the Snowflake connector and utilize native operations to extend your API's capability."
+sidebar_label: Snowflake
+keywords:
+  - snowflake
+  - configuration
+  - connector
+seoFrontMatterUpdated: false
+---
+
+# Snowflake
+
+## Introduction
+
+Hasura DDN includes a Native Data Connector for Snowflake, providing integration with Snowflake instances. This
+connector allows you to leverage Snowflake’s powerful capabilities while taking advantage of Hasura’s metadata-driven
+approach. Here, we’ll explore the key features of the Snowflake connector and walk through the configuration process
+within a Hasura DDN project.
+
+:::info Important Considerations for Using Hasura DDN with Snowflake
```

Comment:
Maybe this admonition should be on the how to build section also?

Reply:
Let me think on this one.